openedx_webhooks/lib/edx_repo_tools_data/models.py

```diff
@@ -125,11 +117,18 @@ def agreement_expires_on(self):
         return expires_on
 
     @property
+    def before(self):
+        """
+        Optional[Dict[str, Dict]]: User's `before` data.
+        """
+        return self._data.get('before')
```

Comment:
I'd rather defer this discussion until later, when we are designing the API to `repo-tools-data`. I'm not sure what the unifying interface is to façade both `before` and `expires_on`, since I'm not aware of all the use cases.

At least in this iteration, you only have to ask `Person.agreement_expired` and `Person.agreement_expires_on`.

We can always make `before` a private property.